client/app/components/_topbar/topbar.controller.js

```diff
@@ -24,6 +25,15 @@ class TopbarController {
     this.$mdSidenav("right").toggle();
   }
 
+  logOut(){
+    this.Authentication.logout()
+      .then(() => {
+        this.$location.hash("/login");
```

Comment:
Actually, i'm just not very familiar with angularJS. Appreciate the edit to include the suggested method 😄